### PR TITLE
15: Ensure transactions saved as they finish rather than in DESTROY

### PR DESCRIPTION
**Pull request by ungrim97**

### CODE REVIEW

**Comment by ungrim97** [comment]
PR to fix issue #15

File: Changes [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {{$NEXT}}
 
+- #15: Fixed issue where save wouldn't work if called during global destruction
```
Comment (PopeFelix) [inline Changes:3]: Mark that as "Issue 15", otherwise folk are going to be wondering what the "15" means. :)

Comment (PopeFelix) [inline Changes:3]: I like this, and it all looks pretty good. However, I'd like to see a unit test that shows all transactions having been recorded without the thing going out of scope.

**Review by PopeFelix — CHANGES_REQUESTED** [review]

**Review by PopeFelix — APPROVED** [review]
Oh, to hell with it. Let's just merge it and hope for the best.

**Comment by PopeFelix** [comment]
In v 1.56

**Pull request merged**